### PR TITLE
N°6922 - Update chinese translations

### PR DESCRIPTION
**Pull request by bdejin**
Chinese translation file updated and tested.

### CODE REVIEW

File: zh_cn.dict.combodo-approval-extended.php [hunk 1]
```diff
@@ -22,20 +22,20 @@
...
-				      <b>Additional information</b>:<br>
-				      <div>$object->html(service_details)$</div>~~',
-	'Approbation:FormBody' => '<p>Dear $approver->html(friendlyname)$, please take some time to approve or reject the ticket</p>~~',
-	'Approbation:ApprovalRequested' => 'Your approval is requested~~',
-	'Approbation:Introduction' => '<p>Dear $approver->html(friendlyname)$, please take some time to approve or reject ticket $object->html(friendlyname)$</p>~~',
+				      <b>附加信息</b>:<br>
+				      <div>$object->html(service_details)$</div>',
+	'Approbation:FormBody' => '<p>尊敬的$approver->html(friendlyname)$, 请您抽空批复工单</p>',
+	'Approbation:ApprovalRequested' => '需要您的审批',
+	'Approbation:Introduction' => '<p>尊敬的$approver->html(friendlyname)$, 请您抽空批复工单$object->html(friendlyname)$</p>~~',
```
Comment (Molkobain) [inline zh_cn.dict.combodo-approval-extended.php:38]:
```suggestion
	'Approbation:Introduction' => '<p>尊敬的$approver->html(friendlyname)$, 请您抽空批复工单$object->html(friendlyname)$</p>',
```

File: zh_cn.dict.combodo-approval-extended.php [hunk 2]
```diff
@@ -45,61 +45,61 @@
...
-	'Class:ApprovalRule/Attribute:level1_default_approval/Value:no' => 'no~~',
-	'Class:ApprovalRule/Attribute:level1_default_approval/Value:no+' => 'no~~',
-	'Class:ApprovalRule/Attribute:level1_default_approval/Value:yes' => 'yes~~',
-	'Class:ApprovalRule/Attribute:level1_default_approval/Value:yes+' => 'yes~~',
-	'Class:ApprovalRule/Attribute:level1_timeout' => 'Level 1 approval delay (hours)~~',
+	'Class:ApprovalRule/Attribute:level1_default_approval/Value:no' => '否',
+	'Class:ApprovalRule/Attribute:level1_default_approval/Value:no+' => '否',
+	'Class:ApprovalRule/Attribute:level1_default_approval/Value:yes' => '是',
+	'Class:ApprovalRule/Attribute:level1_default_approval/Value:yes+' => '是',
+	'Class:ApprovalRule/Attribute:level1_timeout' => '一级审批时限 (小时)~~',
```
Comment (Molkobain) [inline zh_cn.dict.combodo-approval-extended.php:62]:
```suggestion
	'Class:ApprovalRule/Attribute:level1_timeout' => '一级审批时限 (小时)',
```

**Comment by Molkobain** [comment]
Accepted during functional review, will be merged once iTop 3.1.1 packages are released.

**Comment by piRGoif** [comment]
3.1.1 was published, PR is ready to merge

**Pull request merged**